<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>COM_Window</class>
 <widget class="QMainWindow" name="COM_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>738</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>967</width>
    <height>738</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>967</width>
    <height>738</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="BackBtn">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>640</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family:'仿宋';
font-size:20px;
color: #2f3640;
background-color: #eea2a4;
border-color: #eea2a4;
border-radius: 20px;
border-style: solid;
border-width: 2px;
padding: 5px;</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>181</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QComboBox" name="COMcomboBox">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>31</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family:'仿宋';
font-size:20px;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;串口号&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="BitcomboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>31</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family:'仿宋';
font-size:20px;</string>
       </property>
       <property name="currentIndex">
        <number>6</number>
       </property>
       <item>
        <property name="text">
         <string>2400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4800</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>57600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>128000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>230400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>256000</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;波特率&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="DatacomboBox">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>31</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family:'仿宋';
font-size:20px;</string>
       </property>
       <property name="currentIndex">
        <number>3</number>
       </property>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;数据位&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QComboBox" name="StopcomboBox">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>31</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family:'仿宋';
font-size:20px;</string>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;停止位&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QComboBox" name="CRCcomboBox">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>31</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family:'仿宋';
font-size:20px;</string>
       </property>
       <item>
        <property name="text">
         <string>无</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>奇</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>偶</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;校验位&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="ReceiveText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>971</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>281</x>
      <y>482</y>
      <width>379</width>
      <height>170</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="OpenCOMBtn">
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>41</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font-family:'仿宋';
font-size:20px;
color: #2f3640;
background-color: #eea2a4;
border-color: #eea2a4;
border-radius: 20px;
border-style: solid;
border-width: 2px;
padding: 5px;</string>
         </property>
         <property name="text">
          <string>打开串口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="RenewCOMBtn">
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>41</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font-family:'仿宋';
font-size:20px;
color: #2f3640;
background-color: #eea2a4;
border-color: #eea2a4;
border-radius: 20px;
border-style: solid;
border-width: 2px;
padding: 5px;</string>
         </property>
         <property name="text">
          <string>刷新串口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ClearBtn">
         <property name="minimumSize">
          <size>
           <width>121</width>
           <height>41</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font-family:'仿宋';
font-size:20px;
color: #2f3640;
background-color: #eea2a4;
border-color: #eea2a4;
border-radius: 20px;
border-style: solid;
border-width: 2px;
padding: 5px;</string>
         </property>
         <property name="text">
          <string>清除窗口</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font-family:'仿宋';
font-size:20px;</string>
         </property>
         <property name="text">
          <string>输出设置：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="StopShowcheckBox">
         <property name="styleSheet">
          <string notr="true">border: 0px solid gray;   /* 边框 */
border-radius: 4px;   /* 圆角 */
padding: 1px 18px 1px 3px;   /* 字体填衬 */
color: #000;
font: normal normal 20px &quot;仿宋&quot;;
background: transparent;</string>
         </property>
         <property name="text">
          <string>停止显示</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="LogShowcheckBox">
         <property name="styleSheet">
          <string notr="true">border: 0px solid gray;   /* 边框 */
border-radius: 4px;   /* 圆角 */
padding: 1px 18px 1px 3px;   /* 字体填衬 */
color: #000;
font: normal normal 20px &quot;仿宋&quot;;
background: transparent;</string>
         </property>
         <property name="text">
          <string>日志模式</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">font-family:'仿宋';
font-size:20px;</string>
         </property>
         <property name="text">
          <string>输出编码：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="ASCIIradioButton">
         <property name="styleSheet">
          <string notr="true">border: 0px solid gray;   /* 边框 */
border-radius: 4px;   /* 圆角 */
padding: 1px 18px 1px 3px;   /* 字体填衬 */
color: #000;
font: normal normal 20px &quot;仿宋&quot;;
background: transparent;</string>
         </property>
         <property name="text">
          <string>ASCII</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="HEXradioButton">
         <property name="styleSheet">
          <string notr="true">border: 0px solid gray;   /* 边框 */
border-radius: 4px;   /* 圆角 */
padding: 1px 18px 1px 3px;   /* 字体填衬 */
color: #000;
font: normal normal 20px &quot;仿宋&quot;;
background: transparent;</string>
         </property>
         <property name="text">
          <string>HEX</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>967</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
